CHANGELOG.md

```diff
@@ -30,8 +33,6 @@ Update to [Pulumi SDK v3.65.0](https://github.com/pulumi/pulumi/releases/tag/v3.
   image of the same version, fixing [issue
   #411](https://github.com/pulumi/pulumi-kubernetes-operator/issues/411)
   [#422](https://github.com/pulumi/pulumi-kubernetes-operator/pull/422)
-- Added lastUpdate state update to ensure that synchronized commits display a succeeded status.
```

Comment:
Note: this feature was not actually released in v1.11.5 in #429, it should have been nested under HEAD (unreleased), but was missed on PR reveiw.